ownCloudExtApp/Network/DPDownload.m

```diff
@@ -405,6 +405,9 @@ - (void) failureInDownloadProcessWithError:(NSError*)error andResponse:(NSHTTPUR
             case kOCErrorServerPathNotFound:
                 [self.delegate downloadFailed:NSLocalizedString(@"download_file_exist", nil) andFile:self.file];
                 break;
+            case kOCErrorServerMaintenanceError:
+                [self.delegate downloadFailed:NSLocalizedString(@"maintenance_mode_on_server_message", nil) andFile:self.file];
```

Comment:
Where is define the value of @"maintenance_mode_on_server_message"; why is not shown in the diff?